src/store/invitations/actions.ts

```diff
@@ -16,16 +16,16 @@ import { AxiosResponse } from 'axios';
 export const actions: ActionTree<InvitationsStateInterface, RootStateInterface> & Actions = {
   [InvitationsActionsEnum.CREATE_INVITATION]: async ({ commit }, payload) => {
     try {
-      const payloadToSend = new PostInvitationDTOModel(payload);
+      const payloadToSend = new PostInvitationDTOModel(payload.payload);
```

Comment:
Maybe, it would be better to name variable 'data' instead of 'payload' for easier reading. F.e.: 'data.payload'

Reply:
Yes, I'll correct it